src/interfaces.jl

```diff
@@ -254,7 +254,15 @@ end
 function Plot{Func}(user_args::Tuple, user_attributes::Dict) where {Func}
     # Handle plot!(plot, attributes::Attributes, args...) here
     if !isempty(user_args) && first(user_args) isa Attributes
-        merge!(user_attributes, attributes(first(user_args)))
+        # Only forward attributes that are usable by Plot{Func}
+        # TODO, there are two problems with this:
+        # 1) It's a bit magical which attributes are forwarded, when writing a recipe
+        # 2) We can't warn for wrong attributes, since if we do `plot(p, some_other_plot.attributes, ...)`,
```

Comment:
wasn't the warning for wrong attributes the reason why we did this in the first place? I wouldn't like reintroducing that behavior again in a different place..

Reply:
So what's the scenario where someone does that? Is it likely that there are many other invalid attributes in the collection that are annoying to take out manually? One could also filter them by the keys of the recipe to forward to?

Reply:
>One could also filter them by the keys of the recipe to forward to?

That's what we do here, or do you mean something else?